src/main/java/edu/harvard/h2ms/domain/core/Event.java

```diff
@@ -57,6 +57,10 @@
 	@NotNull
 	@Column
     private String location;
+
+	@NotNull
+	@Column
+	private Boolean washed;
```

Comment:
This isn't part of the data model.  Please review the UML diagram, and the stuff I wrote in the Milestone 2 document. There can't be anything handwashing specific in an Event.

An event has an EventTemplate, and the Washed question is part of HandwashingEventTemplate that gets seeded.

The challenge in the reporting is we need a generic way to think about how we generate reports.

Reply:
Oh snap. Ben told me to add ‘washed’. I’ll update this one

Reply:
We already have washed in the event template.